judge/migrations/0110_peg_merge.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.12 on 2020-08-01 03:18
```

Comment:
As suggested elsewhere, this should be moved into a separate `peg_migration` app.